src/utils/application.data.ts

```diff
@@ -46,19 +46,29 @@ import { isActiveFeature } from "./feature.flag";
 import { isNoChangeJourney } from "./update/no.change.journey";
 import { getOverseasEntity } from "../service/overseas.entities.service";
 
-export const getApplicationData = async (session: Session | undefined, req?: Request): Promise<ApplicationData> => {
+export const getApplicationData = async (sessionOrRequest: Session | Request | undefined): Promise<ApplicationData> => {
+
+  const defaultAppData = {};
```

Comment:
Perhaps call this variable `emptyAppData`.

Reply:
Updated.

Reply:
Thanks